src/pages/ReviewPage.tsx

```diff
@@ -326,7 +328,7 @@ class ReviewPage extends React.Component<RouteComponentProps, State> {
                     to={`/contests/${i}#review`}
                   >
                     <ContestProse compact>
-                      <h3 aria-label={`${contest.section}, ${contest.title},`}>
```

Comment:
Seeming like `contest.section` was removed on purpose. I feel bad that visually impaired voters are not getting the same content which sighted voters are getting.